src/dll/Hooks/InitScripts.cpp

```diff
@@ -0,0 +1,75 @@
+#include "InitScripts.hpp"
+#include "Addresses.hpp"
+#include "App.hpp"
+#include "Hook.hpp"
+#include "Systems/ScriptSystem.hpp"
+#include "stdafx.hpp"
+
+namespace
+{
+bool isAttached = false;
+
+void* _CBaseEngine_InitScripts(RED4ext::CBaseEngine* aEngine, unsigned __int8 a2, unsigned __int16 a3);
+Hook<decltype(&_CBaseEngine_InitScripts)> CBaseEngine_InitScripts(Addresses::CBaseEngine_InitScripts,
+                                                                  &_CBaseEngine_InitScripts);
+
+void* _CBaseEngine_InitScripts(RED4ext::CBaseEngine* aEngine, unsigned __int8 a2, unsigned __int16 a3)
+{
+    auto scriptSystem = App::Get()->GetScriptSystem();
+    auto original = aEngine->scriptsBlobPath;
+    if (aEngine->scriptsBlobPath.Length())
+    {
+        spdlog::info("Found scriptsBlobPath: '{}'", aEngine->scriptsBlobPath.c_str());
+        scriptSystem->SetUsingRedmod(true);
+        scriptSystem->SetScriptsBlobPath(&aEngine->scriptsBlobPath);
+        aEngine->scriptsBlobPath = "";
```

Comment:
So here we are forcing a recompilation, right?

Reply:
Yeah, `scriptsBlobPath` is set here when redmod compiles a `modded/final.redscripts` file, so by emptying it, we're tricking the following function into thinking it needs to check for the `EnableCompilation` flag & compile the scripts as usual.

Reply:
Nice!